wristJointController:
  controller:
    package: dynamixel_controllers
    module: joint_position_controller
    type: JointPositionController
  joint_name: wristJoint
  joint_speed: 1.0
  motor:
    id: 0
    init: 512
    min: 0
    max: 512

armJointController:
  controller:
    package: dynamixel_controllers
    module: joint_torque_controller
    type: JointTorqueController
  joint_name: armJoint
  joint_speed: 2.0
  motor:
    id: 1
    init: 0
    min: 0
    max: 0

handJointController:
  controller:
    package: dynamixel_controllers
    module: joint_torque_controller
    type: JointTorqueController
  joint_name: handJoint
  joint_speed: 2.0
  motor:
    id: 2
    init: 0
    min: 0
    max: 0
